Apps/Database/src/Models/ResourceDelegate.cs

```diff
@@ -66,5 +70,17 @@ public class ResourceDelegate : AuditableEntity
         /// Gets or sets the UserProfile associated with <see cref="ProfileHdid"/>.
         /// </summary>
         public virtual UserProfile UserProfile { get; set; } = null!;
+
+        /// <summary>
+        /// Gets or sets the access for the data sets.
+        /// </summary>
+        [Column(TypeName = "jsonb")]
+        [SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "Team decision")]
+        public HashSet<DataSource>? DataSources { get; set; }
```

Comment:
Suggest moving this up above UserProfile so all the navigation properties are at the bottom.

Reply:
Changed.